fire/api/model/punkttyper.py

```diff
@@ -408,6 +462,13 @@ class Koordinat(FikspunktregisterObjekt):
         "Beregning", secondary=beregning_koordinat, back_populates="koordinater"
     )
 
+    tidsserier = relationship(
+        "Tidsserie",
+        order_by="Tidsserie.objektid",
```

Comment:
Jeg har set denne konstruktion flere steder nu, men slår tilfældigt ned her.

Er det korrekt forstået, at sortering efter id er valgt, fordi det her er en fortløbende int og vi derved opnår en kronologisk rækkefølge (efter oprettelse)?

Med andre ord, for UUID-baserede ID'er giver det ikke rigtig mening, da de ikke har nogen rækkefølge indbygget.

Reply:
Ja, det er noget i den stil. Man kunne nok også bruge `registreringfra`. Det er sikkert tilfældigt at det er objektid. Lige her ligger der ikke meget tanke bag det - jeg har kigget på det vi i forvejen havde og lavet noget der lignede.